tests/fast-integration/Makefile

```diff
@@ -30,3 +31,8 @@ ci-post-upgrade:
 	npm install
 	npm run upgrade-test-tests
 	npm run upgrade-test-cleanup
+
+.PHONY: ci-test-eventing
+ci-test-eventing:
+	npm ci
```

Comment:
Do you mean `make ci` here? I don't fine any definition of `npm ci` in package.json

Reply:
https://docs.npmjs.com/cli/v7/commands/npm-ci